### PR TITLE
Miscellaneous improvements to commonjs-extension-resolution-loader and pgp-loader

### PR DESCRIPTION
**Pull request by GeoffreyBooth**
Fixes to get the build passing:

- Restore files accidentally gitignored
- Add `shortCircuit: true` to PGP Loader (needed because we’re building in Node 18 now)

Miscellaneous fixes:

- Use `isBuiltin` per https://github.com/nodejs/loaders-test/pull/10#discussion_r979362861
- Bump version of commonjs-extension-resolution-loader
- Fix misspellings

### CODE REVIEW

File: commonjs-extension-resolution-loader/test/fixtures/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+!node_modules/
```
Comment (arcanis) [inline commonjs-extension-resolution-loader/test/fixtures/.gitignore:1]: Why do you need that, since you have a lockfile?

Comment (GeoffreyBooth) [reply commonjs-extension-resolution-loader/test/fixtures/.gitignore:1]: Those fixtures are fake modules, not packages on npm. Look under `test/fixtures`, it's lots of variations of types of `package.json`.

File: commonjs-extension-resolution-loader/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "commonjs-extension-resolution-loader",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (aduh95) [inline commonjs-extension-resolution-loader/package.json]: Is this on purpose to change the version of `commonjs-extension-resolution-loader` and not `pgp-loader`? In any case, I think it should be reflected on the `package-lock.json`?

Comment (GeoffreyBooth) [reply commonjs-extension-resolution-loader/package.json]: Yeah, I forgot to change the version with the previous changes. We aren’t publishing these and we haven’t really been careful about bumping versions previously, so I’m not sure if I should even change this? Or set it to 0.0.0 and leave it at that?

File: commonjs-extension-resolution-loader/test/fixtures/.gitignore (hunk 1 above)
Comment (arcanis) [reply commonjs-extension-resolution-loader/test/fixtures/.gitignore:1]: Ah that makes sense, lgtm then 👍

**Review by JakobJingleheimer — APPROVED** [review]

**Comment by GeoffreyBooth** [comment]
I’m removing all version bumps from this PR so we can discuss those separately. I’m leaning towards never changing the versions in the `package.json` files since they’re irrelevant as we’re not publishing these loaders anywhere.

**Pull request merged**